docs/csharp/language-reference/configure-language-version.md

```diff
@@ -75,7 +75,7 @@ Builds in all subdirectories of the directory containing that file will use the
 
 ## C# language version reference
 
-The following table shows all current C# language versions. Your compiler may not necessarily understand every value if it's older. If you install .NET Core 3.0 or later, then you have access to everything listed.
+The following table shows all current C# language versions. Your compiler may not necessarily understand every value if it's older. If you install the latest .NET SDK, then you have access to everything listed.
```

Comment:
Seems kind of pointless to be overly specific here and need to update it all the time.